scrapers/LittleCapriceDreams.yml

```diff
@@ -0,0 +1,24 @@
+name: LittleCaprice
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.littlecaprice-dreams.com/project
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //meta[@property="og:title"]/@content
+      Details: //meta[@property="og:description"]/@content
```

Comment:
imo details is more accurate (close to what the site shows) like this
```yaml
      Details: //div[@class="vid_desc"]/p
```
the og:description is more like a synopsis

Reply:
I was debating between og:description and the "vid_desc" class. I'm fine with making the change to the details to what you propose.